### DISCUSSION TITLE
hand over "ASIN" within OPF-FIle ?

### DISCUSSION THREAD

**Discussion by Gru0810** [advplyr/audiobookshelf#1329] [Q&A] [answered]
I use opf-File to transfer Infos to ABS. With this way I'm able to transfer the "ISBN" by using the following line in opf-File:
<dc:identifier id="BookId" opf:scheme="ISBN">9783844541236</dc:identifier>

I tried following line :
<dc:identifier id="BookId" opf:scheme="ASIN">3844541357</dc:identifier>
But ABS did not grab the content.
What is the  best way handover the "ASIN"?

**Comment by advplyr** [comment, marked as answer]
This is not supported for OPF file but I created a feature request for it https://github.com/advplyr/audiobookshelf/issues/1330

**Comment by Gru0810** [comment]
Great! waiting with pleasure for it. thx
